rs/bitcoin/ckbtc/minter/src/address.rs

```diff
@@ -84,6 +84,16 @@ impl BitcoinAddress {
             None => Err(ParseAddressError::NoData),
         }
     }
+
+    pub fn dust_limit(&self) -> Satoshi {
```

Comment:
Note that we didn't add this before because it requires us to update it whenever there is a new type. Granted, this doesn't happen very often but it is simpler to just use the maximum.

Reply:
Ok I replaced the concrete values below with the maximum which is `546`. I think it still makes sense to have this function and an exhaustive match, since adding a new address type will require to check whether `546` is still the maximum dust value. WDYT?

Reply:
Removed  after discussing offline. See the new constant `MINTER_ADDRESS_DUST_LIMIT`